icu4c/source/configure.ac

```diff
@@ -100,6 +100,16 @@ UCONFIG_CPPFLAGS=""
 # such as -std
 UCONFIG_CFLAGS=""
 
+# Check whether to install icu-config
```

Comment:
sadly, our `configure` script actually gets checked into the repo. (Maybe we should change that…) Can you run `autoreconf || (aclocal && autoconf)` ?

Reply:
No problem.